app/models/manageiq/providers/ibm_cloud/inventory/parser/vpc.rb

```diff
@@ -133,6 +135,41 @@ def hardware_networks(persister_hardware, instance)
     end
   end
 
+  def vm_and_template_labels(resource, tags)
+    tags.each do |tag|
+      formatted_tag = tag[:name].split(":")
+      tag_key = tag[:key] || formatted_tag[0] if formatted_tag.length >= 1
+      tag_value = tag[:value] || formatted_tag[1] if formatted_tag.length >= 2
```

Comment:
This looks identical to the logic below https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/91/files#diff-be65168162a51f26d1c4e78ad610a312bea5fbefd1cff9fb19152ef1e89b9992R159-R161

Maybe pull this out into a helper method like `parse_tag_key_value`?